src/app/Validation/ValidationProblemDetailsResult.cs

```diff
@@ -85,6 +83,62 @@ private static string WriteJsonOutput(ActionContext context, ILogger logger)
             return JsonSerializer.Serialize(problemDetails, jsonOptions);
         }
 
+        /// <summary>
+        /// Create a standard validation error
+        /// </summary>
+        /// <param name="key">validation error key</param>
+        /// <returns>standardized ValidationError</returns>
+        private static ValidationError CreateValidationError(string key)
+        {
+            return new ValidationError("InvalidValue", ToCamelCase(key), GetErrorMessage(key));
+        }
+
+        /// <summary>
+        /// Convert key to camel case
+        /// </summary>
+        /// <param name="key">validation error key</param>
+        /// <returns>string</returns>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Globalization", "CA1308:Normalize strings to uppercase", Justification = "tolower is correct")]
+        private static string ToCamelCase(string key)
+        {
```

Comment:
I liked your suggestion of naming this PascalToCamelCase and calling out in a comment/summary that it assumes a valid Pascal input.

Reply:
updated